src/bpmailsend.c

```diff
@@ -5,14 +5,16 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <unistd.h>
+#include <string.h>
+#include <zlib.h>  // for compression
 
 #include "bp.h"
 #include "dtpc.h"
```

Comment:
```suggestion
#include <stdlib.h>
#include <string.h>
#include <unistd.h>

#include "bp.h"
#include "dtpc.h"
#include "zlib.h"
```
Don't need the comment